Classes/People/PeopleViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  PeopleViewController.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 11/19/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+protocol PeopleViewControllerDelegate: class {
+    func didDismiss(controller: PeopleViewController, selections: [User])
+}
+
+final class PeopleViewController: UITableViewController {
+
+    enum PeopleType {
+        case assignee
+        case reviewer
+    }
+
+    private let selectionLimit = 10
+    private weak var delegate: PeopleViewControllerDelegate?
+    private var owner: String!
+    private var repo: String!
+    private var client: GithubClient!
+    private var users = [User]()
+    private let feedRefresh = FeedRefresh()
+    private var type: PeopleType!
+    private var selections = Set<String>()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.refreshControl = feedRefresh.refreshControl
+        feedRefresh.refreshControl.addTarget(self, action: #selector(LabelsViewController.onRefresh), for: .valueChanged)
+
+        feedRefresh.beginRefreshing()
+        fetch()
+        updateSelectionCount()
+    }
+
+    // MARK: Private API
+
+    func updateSelectionCount() {
+        let label = UILabel()
+        label.font = Styles.Fonts.body
+        label.backgroundColor = .clear
+        label.textColor = Styles.Colors.Gray.light.color
+        label.text = "\(selections.count)/\(selectionLimit)"
+        label.sizeToFit()
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: label)
+    }
+
+    @objc func onRefresh() {
+        fetch()
+    }
+
+    func fetch() {
+        client.fetchAssignees(owner: owner, repo: repo) { [weak self] result in
+            self?.feedRefresh.endRefreshing()
+            switch result {
+            case .success(let users):
+                self?.users = users
+                self?.tableView.reloadData()
+            case .error:
+                ToastManager.showGenericError()
+            }
+        }
+    }
+
+    @IBAction func onDone(_ sender: Any) {
+        let selections = users.filter { self.selections.contains($0.login) }
+        delegate?.didDismiss(controller: self, selections: selections)
+        dismiss(animated: true)
+    }
+
+    // MARK: Public API
+
+    func configure(
+        selections: [String],
+        type: PeopleType,
+        client: GithubClient,
+        delegate: PeopleViewControllerDelegate,
+        owner: String,
+        repo: String
+        ) {
+        self.selections = Set<String>(selections)
```

Comment:
Not sure where the best place to make the change would be (maybe an option could get passed through to the `configure` method -- but we should always have the signed-in user as the first option to quickly assign yourself!

Reply:
GREAT idea. I'll look into adding a "Claim" button at the very top